src/app/payment/paymentMethod/PaymentMethodTitle.tsx

```diff
@@ -51,7 +53,7 @@ function getPaymentMethodTitle(
             },
             [PaymentMethodId.PaypalCommerceAlternativeMethod]: {
                 logoUrl: method.logoUrl || '',
-                titleText: (method.logoUrl ? '' : method.config.displayName) || '',
+                titleText: method.logoUrl ? '' : methodDisplayName,
```

Comment:
We can take the logo url from method object and for this particular case we check if we have the logo we shows logo, otherwise we will show display name here. But it is a part of the old logic and I just changed method.config.displayName to methodDisplayName  what made the same under the hood